docs/references/files/fossa-deps.md

```diff
@@ -91,6 +91,48 @@ vendored-dependencies:
 
 For more details, please refer to the [feature](../../features/vendored-dependencies.md) walk through.
 
+### `fork-aliases:`
+
+Denotes mapping of fork dependencies to their base dependencies. This is useful when you have forked a dependency and want it to be treated as the original dependency by FOSSA. This, for example, will allow FOSSA to find and report security issues that are associated with the root project.
+
+- `fork`: The fork dependency entry that should be aliased to the base dependency. (Required)
+  - `type`: Type of the fork dependency. (Required)
+  - `name`: Name of the fork dependency. (Required)
+  - `version`: Version of the fork dependency. (Optional)
+- `base`: The base/original dependency entry that your fork should be aliased to. (Required)
+  - `type`: Type of the base dependency. (Required)
+  - `name`: Name of the base dependency. (Required)
+  - `version`: Version of the base dependency. (Optional)
+- `labels`: An optional list of labels to be added to the fork alias.
+
+**Version Matching rules:**
+- If `fork` version is specified, only that exact version will be translated
+- If `fork` version is not specified, any version will match
```

Comment:
Does this mean that every vulnerability for any version of the upstream package gets reported?